Makefile.toml

```diff
@@ -0,0 +1,40 @@
+[env]
```

Comment:
Can you explain a bit why the need for the this new `Makefile.toml` file? How is it different from a traditional Makefile?

Reply:
so its a part of this cargo make plug in, it kind of augments the make file. Use it mostly for convenience cargo make run, cargo make int-tests etc